webpack/JobWizard/JobWizard.js

```diff
@@ -169,8 +169,23 @@ export const JobWizard = ({ rerunData }) => {
                   job_template: { name, description_format },
                 },
               }) => {
+                const allowedInputs = template_inputs
+                  .map(({ inputName }) => inputName)
+                  .concat(
+                    advanced_template_inputs.map(({ inputName }) => inputName)
```

Comment:
`template_inputs` and `advanced_template_inputs` dont have a key named `inputName`
I assume you meant to do here something like this? (name is already defined so need to use _name)
```suggestion
                  .map(({ name: _name }) => _name)
                  .concat(
                    advanced_template_inputs.map(({ name: _name }) => _name)
```

Reply:
Yes, of course. I originally used something like `({ name } => name`, got yelled at by the linter for shadowing an outer variable so I just renamed things without thinking, sorry about that